test/functional/android/appium_tests.py

```diff
@@ -131,7 +131,7 @@ def test_pull_folder(self):
     def test_background_app(self):
         self.driver.background_app(1)
         sleep(5)
-        el = self.driver.find_element_by_name('Animation')
```

Comment:
name locators have been replaced with accessibility id

Reply:
Ok, got it. Thanks.